src/ReactSVG.tsx

```diff
@@ -164,7 +174,11 @@ export class ReactSVG extends React.Component<Props, State> {
     const Wrapper = wrapper!
 
     return (
-      <Wrapper {...rest} ref={this.refCallback}>
+      <Wrapper
+        {...rest}
+        ref={this.refCallback}
+        {...(wrapper === 'svg' ? { xmlns: 'http://www.w3.org/2000/svg' } : {})}
```

Comment:
Hmm, don't know whether to leave this here and remove the `setAttribute` farther up the file, or do the opposite. If I leave this here, the attribute gets added to the snapshot in the test, but not the other way around.

Need to dig a bit more into it but just heading out the door, so will catch up on things when back 🏃🏽